controller/files.go

```diff
@@ -74,6 +84,33 @@ func (f *Files) DeleteFile(scriptID, projectID uuid.UUID) error {
 		return errors.Wrap(err, "failed to delete cadence file")
 	}
 
+	err = f.fileChanged(projectID)
+	if err != nil {
+		return errors.Wrap(err, "failed to update project on file change")
+	}
+
+	return nil
+}
+
+// fileChanged updates project when a file is changed to set new updated time
+func (f *Files) fileChanged(projectID uuid.UUID) error {
+	var proj model.Project
+	err := f.store.GetProject(projectID, &proj)
+	if err != nil {
+		return err
+	}
+
+	err = f.store.UpdateProject(model.UpdateProject{
+		ID:          proj.ID,
+		Title:       nil,
+		Description: nil,
+		Readme:      nil,
+		Persist:     nil,
```

Comment:
Why is this set to nil? I feel you should just skip all these. Also are you doing this to just update the updated_at time? A comment as to why this is done is really needed here.

Reply:
Cleaned this up and added better comments